device-types/HPE/Aruba-EC-US.yaml

```diff
@@ -1,20 +1,21 @@
 ---
 manufacturer: HPE
-model: Aruba ec-us
+model: Aruba EC-US
 slug: hpe-aruba-ec-us
 part_number: '201106'
 u_height: 1
 is_full_depth: false
 airflow: passive
+weight: 1.1
```

Comment:
Looks like this file is missing the image properties
`front_image: true`
`rear_image: true`